skimage/transform/steerable.py

```diff
@@ -0,0 +1,406 @@
+from __future__ import division
+import numpy as np
+from .._shared.utils import assert_nD
+from ..util import img_as_float
+
+
+def build_steerable(image, height=5, nbands=4):
+    """ Construct a Steerable subband decomposition of a gray scale image.
+
+    Parameters
+    ----------
+    image : 2-D array
+        input image
+    height : integer, optional
+        Height of the steerable decomposition. This includes the counting of
+        low pass and high pass subbands.
+    nbands : integer, optional
+        Number of orientations in the Steerable decomposition.
+
+    Returns
+    -------
+    coeff : list of lists of numpy array
+        subbands of Steerable decomposition,
+        stored as a list of 'height' sublists,
+        Sublists correspond to decreasing radius level in Steerable pyramid
+        The first sublist contains the high pass subband.
+        The last sublist contains the low pass subband.
+        Intermediate sublists contain subbands from different orientations.
+
+    References
+    ----------
+    .. [1] E. P. Simoncelli and W. T. Freeman
+        "The Steerable Pyramid: A Flexible Architecture
+        for Multi-Scale Derivative Computation."
+        http://www.cns.nyu.edu/~eero/steerpyr/
+    """
```

Comment:
please expand this reference here (and at top of the module) to include the publication details:
```
    .. [1] Simoncelli, E.P. & Freeman, W.T.
       (1995). The Steerable Pyramid: A Flexible Architecture for Multi-Scale
       Derivative Computation. In Proc. 2nd IEEE International Conf. on Image
       Proc., vol.III pp. 444-447, Oct 1995. 
       http://www.cns.nyu.edu/~eero/steerpyr/,
       http://www.cns.nyu.edu/pub/eero/simoncelli95b.pdf,
       DOI:10.1109/ICIP.1995.537667
```

Reply:
Done!